authors/apps/articles/views.py

```diff
@@ -99,3 +99,140 @@ def retrieve(self, request, slug):
         serializer = self.detailSerializer()
         serializer.instance = article_instance
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+class RetrieveArticleWithReadingTime(generics.RetrieveAPIView):
+
+    def get_queryset(self):
+
+        queryset = Article.objects.all()
+
+        return queryset
+
+    def retrieve(self, *args):
+
+        articles = self.get_queryset()
+
+        article_body = []
+
+        title = {}
+
+        description = {}
+
+        body = {}
+
+        time = {}
+
+        created_at = {}
+
+        WPM = 200
```

Comment:
For maintainability purposes, since `WPM` is more likely to change, it would be easier if it was in  the settings file.